src/views/ProfileMenu/index.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import {
+  Avatar, IconButton, Menu, MenuItem, Typography,
+} from '@material-ui/core';
+
+const styles = theme => ({
+  root: {
+    display: 'flex',
+  },
+  grow: {
+    marginRight: theme.spacing.unit * 15,
+  },
+  avatar: {
+    width: 25,
+    height: 25,
+    marginRight: theme.spacing.unit * 15,
```

Comment:
Remove this.

Reply:
Thanks, I removed the last line about `marginRight`.